ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -36,27 +33,21 @@ export const AccountCreationSurvey = withProfileErrorModal({title: 'Error creati
   async createAccount(profile, captchaToken): Promise<Profile> {
     const {invitationKey, termsOfServiceVersion, onComplete} = this.props;
 
-    try {
-      const newProfile = await profileApi().createAccount({
-        profile: profile,
-        captchaVerificationToken: captchaToken,
-        invitationKey: invitationKey,
-        termsOfServiceVersion: termsOfServiceVersion
-      });
-      onComplete(newProfile);
-      return newProfile;
-    } catch (error) {
-      reportError(error);
-      const errorResponse = await convertAPIError(error);
-      this.props.showProfileErrorModal(errorResponse.message);
-    }
+    const newProfile = await profileApi().createAccount({
+      profile: profile,
+      captchaVerificationToken: captchaToken,
+      invitationKey: invitationKey,
+      termsOfServiceVersion: termsOfServiceVersion
+    });
+    onComplete(newProfile);
+    return newProfile;
   }
 
   render() {
     return <React.Fragment>
```

Comment:
Is this Fragment necessary?

Reply:
Good catch. Doesn't look like it to me and since we're touching this we might as well remove.